extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -116,6 +118,9 @@ export class AmpStoryPage extends AMP.BaseElement {
 
     /** @private {!Array<function()>} */
     this.unlisteners_ = [];
+
+    /** @private {?./amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
```

Comment:
I think you can use the synchronous method here, since we know the service has been registered by the ``amp-story``.

````
/** @private @const {!./amp-story-store-service.AmpStoryStoreService} */
this.storeService_ = Services.storyStoreService(this.win_);
````

Reply:
I'm not sure if that's always true?  The construction will get involved when the custom element gets created, but I don't know if it's always guarantees that the story will before its pages.